chore : 충돌 해결하며 누락된 코드 반영

higukang opened the pull request:
# 🚀 Pull Request

chore : 충돌 해결하며 누락된 코드 반영

## #️⃣ 연관된 이슈

#435

## 📋 작업 내용

chore : 충돌 해결하며 누락된 코드 반영

higukang committed:
chore : 충돌 해결하며 누락된 코드 반영

oowtl commented on module-service/src/main/java/com/checkping/service/member/MemberServiceImpl.java:
```diff
@@ -260,12 +265,17 @@ public void activateMember(Long memberId) {
         memberRepository.save(member);
+
+        // Redis 1번 저장소에 비활성화된 회원 ID 삭제
+        if(redisConnectionCheckService.isRedisAvailable()){
```
localhost 일 때, isRedisAvailable 를 바로 중단하도록 변경 부탁드려도 될까요?

oowtl reviewed (approved):
충돌 해결 확인하였습니다.

higukang committed:
refactor: Redis 연결 확인 로직 단순화

- `RedisTemplate`을 이용한 `PING` 검사 방식 제거
- `@Value`를 사용하여 `spring.data.redis.host` 값을 주입받도록 변경
- Redis 호스트가 `localhost`일 경우 `false`를 반환하도록 로직 단순화

#Issue: 435
